core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -486,9 +491,11 @@ class SparkContext(config: SparkConf) extends Logging {
       value <- Option(System.getenv(envKey)).orElse(Option(System.getProperty(propKey)))} {
       executorEnvs(envKey) = value
     }
+
```

Comment:
Likewise whitespace changes just aren't important and have a tiny cost (merge conflicts)